tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -282,8 +290,14 @@ EOF
 	# that does not exist.
 	#
 	echo "::group::Check the service healthy"
-	kbs_ip=$(kubectl get -o jsonpath='{.spec.clusterIP}' svc "$KBS_SVC_NAME" -n "$KBS_NS" 2>/dev/null)
-	kbs_port=$(kubectl get -o jsonpath='{.spec.ports[0].port}' svc "$KBS_SVC_NAME" -n "$KBS_NS" 2>/dev/null)
+	if [ "${KBS_INGRESS}" = "nodeport" ]; then
+		kbs_ip=$(kubectl get nodes -o jsonpath='{.items[0].status.addresses[0].address}')
+		kbs_port=$(kubectl get svc kbs -n coco-tenant -o jsonpath='{.spec.ports[0].nodePort}')
```

Comment:
This should use the `$KBS_SVC_NAME` and `$KBS_NS`

Reply:
changes applied